.classpath

```diff
@@ -5,6 +5,10 @@
 	<classpathentry kind="var" path="networktables" sourcepath="networktables.sources"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar" sourcepath="USERLIBS_DIR/navx_frc-sources.jar"/>
-	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
+        <classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar" sourcepath="USERLIBS_DIR/CTRLib-javadoc.jar"/>
```

Comment:
The NavX javadoc is some kind of equilibrium that is reached in Eclipse. It will happen after a certain amount of time.